libs/ledgerjs/packages/types-live/src/feature.ts

```diff
@@ -229,6 +230,9 @@ export type Features = CurrencyFeatures & {
   lldNetworkBasedAddAccount: DefaultFeature;
   llmOfacGeoBlocking: DefaultFeature;
   lldOfacGeoBlocking: DefaultFeature;
+  lldSessionReplay: Feature<{
+    sampling: number;
+  }>;
```

Comment:
You can type it like that 

`export type Feature_LldSessionReplay = Feature<{
  sampling: number;
}>;`